indigo/bulk_creator.py

```diff
@@ -334,6 +336,11 @@ def validate_row(self, view, row):
         form = self.get_row_validation_form(row_data)
```

Comment:
You could delegate all the validation to the RowValidationFormBase. It would need the country to be passed to the form constructor (which would be useful anyway). You can then make doctype a ChoiceField, and in the form's constructor update the `choices` attribute as appropriate. This is what the form is there to do, after all.